x-pack/legacy/plugins/graph/public/angular/templates/index.html

```diff
@@ -7,7 +7,8 @@
     <graph-inspect ng-show="menus.showInspect"></graph-inspect>
   </div>
 
-  <graph-app
+  <div
```

Comment:
Safari and IE don't like the custom elements (like `graph-app`) in the DOM - by using attribute directives here they behave like regular divs.